redfish-core/lib/chassis.hpp

```diff
@@ -147,9 +147,10 @@ inline void handleChassisCollectionGet(
     asyncResp->res.jsonValue["@odata.id"] = "/redfish/v1/Chassis";
     asyncResp->res.jsonValue["Name"] = "Chassis Collection";
 
```

Comment:
There seem more changes in this file.

https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59537/12/redfish-core/lib/chassis.hpp#b104

Reply:
We don't have the same changes in open-bmc/bmcweb, please put the refernce in ibm-openbmc repo.

Reply:
which changes are missed? Put a line number or reference in ibm-openbmc/bmcweb